src/com/couchbase/tools/performer/BuildPerformer.groovy

```diff
@@ -48,6 +48,12 @@ class BuildPerformer {
         else if (sdkRaw == "go") {
             BuildDockerGoPerformer.build(env, dir, version, imageName, onlySource)
         }
+        else if (sdkRaw == "python") {
+            BuildDockerPythonPerformer.build(env, dir, version, Optional.empty(), imageName, onlySource)
+        }
+        else if (sdkRaw == "c++") {
```

Comment:
Should I make it also accept `cpp` and `cxx` here?

Reply:
I'd say use one and use consistently.  'Teach' people to expect "C++" all over the performance project.